lib/ansible/modules/unarchive.py

```diff
@@ -311,7 +311,10 @@ def _permstr_to_octal(self, modestr, umask):
     def _legacy_file_list(self):
         rc, out, err = self.module.run_command([self.cmd_path, '-v', self.src])
         if rc:
-            raise UnarchiveError('Neither python zipfile nor unzip can read %s' % self.src)
+            msg = 'Neither python zipfile nor unzip can read {src}.'.format(src=self.src)
+            if self.module._verbosity >= 3:
```

Comment:
Is it bad to use the private `_verbosity` attribute?

Reply:
Rather than take this approach, I will open a separate PR using `module.log()` to capture the command output.